lib/ach_client/providers/soap/ach_works/date_formatter.rb

```diff
@@ -26,7 +26,7 @@ def self.format(date)
       end
 
       private_class_method def self.format_date(date)
-        date.strftime('%Y-%m-%dT%H:%M:%S.%5N%:z')
+        date.strftime('%Y-%m-%dT%H:%M:%S')
```

Comment:
They don't expect _any_ timezone info? Seems kinda weird to me, but if that's what their API expects… 🤷